### PR TITLE
Exclude OpenJ9 management extensions from compact profiles 1 & 2

### PR DESCRIPTION
**Pull request by keithc-ca**
Fixes https://github.com/ibmruntimes/openj9-openjdk-jdk8/issues/155.

### CODE REVIEW

File: closed/make/CreateJars.gmk [hunk 1]
```diff
@@ -86,22 +86,99 @@ RT_JAR_EXCLUDES += \
...
+	com/ibm/jvm/dtfjview \
+	com/ibm/jvm/format \
+	com/ibm/jvm/j9 \
+	com/ibm/jvm/trace \
+	com/ibm/jvm/traceformat \
 	com/ibm/tools/attach/attacher \
 	#
+
+OPENJ9_MANAGEMENT_TYPES := \
+	com/ibm/java/lang/management/internal/BufferPoolMXBeanImpl.class \
```
Comment (pshipton) [inline closed/make/CreateJars.gmk]: Can we use wildcards to include entire packages? A hard code list of classes will likely result in problems when new classes are added.

Comment (keithc-ca) [reply closed/make/CreateJars.gmk]: Yes, we ought to be do that.

**Comment by keithc-ca** [comment]
Updated as suggested.

**Comment by pshipton** [comment]
jenkins test sanity zlinux

**Pull request merged**